napalm_ios/ios.py

```diff
@@ -890,6 +890,7 @@ def get_facts(self):
                 _, os_version = line.split("IOS (tm) ")
                 os_version = os_version.strip()
 
+        domain_name = 'not set'
```

Comment:
I think we set empty string on the other operating systems.

Reply:
I think to be consistent with the other code here domain_name should be set to 'Unknown' in line 864. Though that might be inconsistent with other platforms do (i.e. with what the default values should be.

Reply:
Well I did the change, but I've no idea why the coverage increased by 2.4% ! Maybe I did something wrong.